simulator/src/window.rs

```diff
@@ -51,17 +51,17 @@ pub enum SimulatorEvent {
         /// The directionality of the scroll (normal or flipped)
         direction: MouseWheelDirection,
     },
+    /// An exit event
+    Quit,
 }
 
 /// Simulator window
 pub struct Window {
     scale: usize,
     pixel_spacing: usize,
     theme: BinaryColorTheme,
-
```

Comment:
This empty line was intentional, to separate the SDL fields from the other fields.